src/api-service/__app__/onefuzzlib/pools.py

```diff
@@ -722,6 +727,13 @@ def cleanup_nodes(self) -> bool:
                 else:
                     to_reimage.append(node)
 
+        idle_nodes_ids = NodeHeartbeat.get_dead_nodes(
+            [node_id for node_id in azure_nodes.keys()], datetime.timedelta(hours=1)
```

Comment:
The `hours=1` determines what counts as "dead", right? Probably worth giving a constant in this module.